games/sudoku-solver/Sudoku.java

```diff
@@ -0,0 +1,53 @@
+import java.util.*;
+class Sudoku {
+  public void sudokuSolver(int[][] board) {
```

Comment:
Do not pass board from the Main. Let it be part of the this class.  Random initialization of the board should be part of this class.

Reply:
e18c92a